packages/contracts/src/contracts/current/protocol/Exchange/Exchange.sol

```diff
@@ -34,13 +34,11 @@ contract Exchange is
 
     function Exchange(
         IToken _zrxToken,
-        ITokenTransferProxy _tokenTransferProxy
-    )
+        ITokenTransferProxy _tokenTransferProxy)
```

Comment:
Let's keep the old style of indentation

Reply:
The new indentation is consistent with all of our other contracts.